assets/settings/default.json

```diff
@@ -300,7 +300,8 @@
       // 1. "gpt-3.5-turbo"
       // 2. "gpt-4"
       // 3. "gpt-4-turbo-preview"
-      "default_model": "gpt-4-turbo-preview"
+      // 4. "gpt-4o"
+      "default_model": "gpt-4o"
```

Comment:
Should we really change the default model?

Reply:
Yeah, I've debated this myself. My thinking was, it's equally capable as 4-turbo, but faster and cheaper. Feels like a nice win for people just using the default config.

Reply:
Makes sense, yeah.